c7n/resources/resource_map.py

```diff
@@ -41,6 +41,7 @@
     "aws.config-recorder": "c7n.resources.config.ConfigRecorder",
     "aws.config-rule": "c7n.resources.config.ConfigRule",
     "aws.customer-gateway": "c7n.resources.vpc.CustomerGateway",
+    "aws.control-panel": "c7n.resources.route53.ControlPanel",
```

Comment:
the resource name feels a little too generic, perhaps arc-control-panel or recovery-control-panel or readiness-control-panel the later would align to the extant resource already for this service.